src/main/java/nextstep/courses/domain/CoverImage.java

```diff
@@ -0,0 +1,16 @@
+package nextstep.courses.domain;
+
+public class CoverImage {
+
+    Long id;
+    String imagePath;
+    String fileName;
+    Long size;
```

Comment:
```suggestion
    private final Long size;
```
인스턴스 변수는 가능하면 private final로 구현하는 것은 어떨까?